File: src/main/java/com/cartobucket/auth/routes/mappers/SchemaRequestFilterMapper.java
package com.cartobucket.auth.routes.mappers;

import com.cartobucket.auth.model.generated.SchemaRequestFilter;

import java.util.List;
import java.util.UUID;

public class SchemaRequestFilterMapper {
    public static SchemaRequestFilter to(List<UUID> authorizationServerIds) {
        var filter = new SchemaRequestFilter();
        filter.setAuthorizationServerIds(authorizationServerIds);
        return filter;
    }
}

File: auth-api/src/main/kotlin/com/cartobucket/auth/api/server/routes/mappers/SchemaRequestFilterMapper.kt
// (C)2024
package com.cartobucket.auth.api.server.routes.mappers

import com.cartobucket.auth.api.dto.SchemaRequestFilter
import java.util.UUID

object SchemaRequestFilterMapper {
    fun to(authorizationServerIds: List<UUID>): SchemaRequestFilter =
        SchemaRequestFilter(
            authorizationServerIds = authorizationServerIds,
        )
}
